src/Request/RequestHandler.php

```diff
@@ -428,6 +441,9 @@ protected function sortParameters(array $parameters)
 
     /**
      * Escapes all special symbols for query
+     * All parameter names and values are escaped using the percent-encoding (%xx) mechanism.
+     * Characters not in the unreserved character set ([RFC3986] section 2.3) MUST be encoded.
+     * Characters in the unreserved character set MUST NOT be encoded.
```

Comment:
As this is a potential attack vector we need to reject 'evil' characters here, like `>` and `}` and others that will never be needed for an oauth signature.

It looks like the only non-alphanumeric characters we require is `/` `:` `.` `+` `,` and `=`, so we should check and throw an exception here if we encounter anything other than these characters, and [a-zA-Z0-9]